src/buttons/__tests__/Button.js

```diff
@@ -43,6 +43,18 @@ describe('Button Component', () => {
     expect(onPress).toHaveBeenCalled();
   });
 
+  it('should not be call onPress events when loading is true', () => {
+    const onPress = jest.fn();
+    const wrapper = shallow(<Button theme={theme} loading={true} />);
+
+    wrapper.setProps({ onPress });
```

Comment:
The onPress as well. Just it on the button.

The only time you'll need to use setProps is when you're testing before and after states.